### PR TITLE
feat: configure theme for home page with font

### PR DESCRIPTION
**Pull request by adrife2005**
### Description
I added custom styles from UiReact to use in our web page, and also import the font-family we're gonna be using.

### How to test?
You can run in the terminal: `pnpm run dev --filter=@1xdev/homepage` to see the web page runing with the syles and the font-family

### Screenshots
image
image

#### After

### CODE REVIEW

**Comment by inavac182** [comment]
Let's breakdown this PR. I see multiple things going on:

- Clean up old package
- Adding pages structure
- Adding custom theme styles

Those could be there own PR.

File: apps/homepage/src/app/contact/contact-form.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function ContactForm() {
+
```
Comment (inavac182) [inline apps/homepage/src/app/contact/contact-form.tsx]: What is this returning? And where is it being used?

File: apps/lab/src/app/global.css [hunk 2]
```diff
@@ -1,3 +1 @@
-@import url('../../node_modules/@uireact/foundation/dist/index.css');
```
Comment (inavac182) [inline apps/lab/src/app/global.css:1]: What is the theme being rendered in the lab app?

Comment (inavac182) [reply apps/lab/src/app/global.css:1]: Looks like there is no theme there:
Screenshot 2024-12-17 at 7 13 58 PM

File: apps/homepage/src/app/layout.tsx [hunk 3]
```diff
@@ -16,7 +18,12 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
-      <body className={`${sen.variable}`}>{children}</body>
+      <body className={`${firaCode.className}`}>
+        <ViewWrapper>
```
Comment (inavac182) [inline apps/homepage/src/app/layout.tsx:22]: Looks like the theme is not changing to dark on the home page app: 

Screenshot 2024-12-17 at 7 15 20 PM

File: apps/lab/src/app/global.css (hunk 2 above)
Comment (adrife2005) [reply apps/lab/src/app/global.css:1]: My bad, i'm gonna add again the styles

**Review by inavac182 — APPROVED** [review]

**Review by iNavac03 — APPROVED** [review]

**Pull request merged**